internal/schema/1.7/removed.go

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: MPL-2.0
+
+package schema
+
+import (
+	"github.com/hashicorp/hcl-lang/lang"
+	"github.com/hashicorp/hcl-lang/schema"
+	"github.com/hashicorp/terraform-schema/internal/schema/refscope"
+	"github.com/zclconf/go-cty/cty"
+)
+
+func removedBlock() *schema.BlockSchema {
+	return &schema.BlockSchema{
+		Description: lang.Markdown("Declaration to specify what address to remove from the state"),
+		Body: &schema.BodySchema{
```

Comment:
```suggestion
		HoverURL: "https://developer.hashicorp.com/terraform/language/resources/syntax#removing-resources"
		Body: &schema.BodySchema{
```

Current URL is https://developer.hashicorp.com/terraform/language/v1.7.x/resources/syntax#removing-resources until the final release is out

There is also https://developer.hashicorp.com/terraform/language/v1.7.x/modules/syntax#removing-modules which could be relevant but we cannot include two URLs and I think one is still better than none?

@kmoe asked in https://github.com/hashicorp/terraform/pull/34339#issue-2020697394 

> Should there be a single page describing the syntax for the removed block somewhere? The IA of the docs is a little confusing to me, as some configuration constructs are given single-page manuals, while others appear only in how-to articles.

and I cannot agree more. Either way if there is such a page in the future, we can still update the URL/schema.